go/vt/vttablet/tabletserver/schema/engine.go

```diff
@@ -343,7 +343,7 @@ func (se *Engine) reload(ctx context.Context) error {
 		// TODO(sougou); find a better way detect changed tables. This method
 		// seems unreliable. The endtoend test flags all tables as changed.
 		tbl, isInTablesMap := se.tables[tableName]
-		if isInTablesMap && createTime < se.lastChange {
+		if isInTablesMap && createTime == tbl.CreateTime && !(createTime >= se.lastChange) {
```

Comment:
`createTime < se.lastChange` looks same as `!(createTime >= se.lastChange)`, can we keep the earlier check, looks easier to read.